pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Programming Language :: Python :: 3",]
+description = "Ophyd Device definitions for LCLS Beamline components"
+dynamic = [ "version", "readme", "dependencies", "optional-dependencies",]
+keywords = []
+name = "pcdsdevices"
+requires-python = ">=3.9"
+
+[options]
+zip_safe = false
+include_package_data = true
```

Comment:
One oddity in the build output that might be worth noting:

<details>

```
packages/setuptools/command/build_py.py:[202](https://github.com/pcdshub/pcdsdevices/actions/runs/3905733272/jobs/6673094192#step:11:203): SetuptoolsDeprecationWarning:     Installing 'pcdsdevices.tests' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'pcdsdevices.tests' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'pcdsdevices.tests' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'pcdsdevices.tests' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.


  !!

    check.warn(importable)
  /home/runner/micromamba/conda-bld/pcdsdevices_1673554832717/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.11/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'pcdsdevices.tests.test_lens_sets' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'pcdsdevices.tests.test_lens_sets' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'pcdsdevices.tests.test_lens_sets' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'pcdsdevices.tests.test_lens_sets' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.


  !!

    check.warn(importable)
  /home/runner/micromamba/conda-bld/pcdsdevices_1673554832717/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.11/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'pcdsdevices.ui' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'pcdsdevices.ui' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'pcdsdevices.ui' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'pcdsdevices.ui' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.

  ```

</details>

Reply:
I think I've resolved it. Good catch! See 48ed02cbb1